### PR TITLE
WIXBUG4847 - jchoover: Show the status in the title area.  Also fixed…

### PR DESCRIPTION
**Pull request by jchoover**
… the news feed image.

### CODE REVIEW

File: src/Setup/WixBA/RootView.xaml [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  <copyright file="RootView.xaml" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
```
Comment (rseanhall) [inline src/Setup/WixBA/RootView.xaml]: Need to keep the .NET Foundation copyright.

Comment (jchoover) [inline src/Setup/WixBA/RootView.xaml]: Was not intentional, is that VS15 removing it cause it's a comment?

**Comment by jchoover** [comment]
Sorry for the force push, fixed the copyright, didn't think 2 XML file changes deserved MD update.

**Comment by robmen** [comment]
All bugs fixes should have History entry. Also nothing wrong with a force push in your fork to keep a clean set of commits in the PR.

**Pull request closed without merging**